doc/data/messages/b/bad-builtin/good.py

```diff
@@ -0,0 +1,2 @@
+numbers = [lambda x: 2 * x, [1, 2, 3]]
+print(numbers)
```

Comment:
```suggestion
numbers = [2 * x for x in [1, 2, 3]]
print(numbers)
```

Reply:
thank you. It was stupid ctr+c + ctrl+v error :-/